conf/portal.application.conf

```diff
@@ -187,14 +187,14 @@ hostProvider {
 # Alerts
 # ~~~~~
 alertRepository {
-  type = com.arpnetworking.metrics.portal.alerts.impl.NoAlertRepository
+  type = com.arpnetworking.metrics.portal.alerts.impl.DatabaseAlertRepository
```

Comment:
My preference at the moment on both counts is no. Alerts one could argue, but they tend to be very organization specific. Expressions just plain don't work (yet).